config/locales/es.yml

```diff
@@ -18,7 +20,15 @@ es:
       confirmation_resent: "No se pudo reenviar la confirmación"
   mailers:
     confirmation_instructions:
-      hello: Hola
-      body: ¡Bienvenidas a Pie for Providers! Prepárate para aumentar tu porción del pastel con herramientas sencillas de administración del subsidio. Para empezar, por favor verifica tu dirección de correo haciendo clic en el enlace abajo (también puedes copiar y pegar en su navegador).
```

Comment:
I think this translation shouldn't be changed back to !Bienvenido - the !Bienvenidas is directly from Chelsea, I think.  See resolved comments (last 3 I think) on #208

Reply:
Fixed.